_posts/2018-10-23-gsoc-setup-assistant-v2.markdown

```diff
@@ -6,7 +6,7 @@ layout: post
 slug: gsoc-2018-setup-assistant-v2
 title: Google Summer of Code Improvements to Setup Assistant
 media_type: image
-media_link: http://moveit.ros.org/assets/images/gsoc.png
+media_link: /assets/images/moveit_setup_assistant.png
```

Comment:
Could this be uploaded to assets/images/gsoc18 to avoid cluttering the images folder? Or do you anticipate the graphic being used elsewhere?

Reply:
Great point, done.